cfssl.tf

```diff
@@ -1,3 +1,27 @@
+data "template_file" "cfssl" {
+  template = <<EOF
+{
+  "ignition": {
+    "version": "2.2.0",
+    "config": {
+      "replace": {
+        "source": "s3://${aws_s3_bucket.userdata.id}/cfssl-config-${sha1(var.cfssl_user_data)}.json",
+        "aws": {
+          "region": "${var.region}"
+        }
+      }
+    }
+  }
+}
```

Comment:
What do you think about putting this template into a single file, with the source object and the region as template variables?

Reply:
sounds good